### ISSUE TITLE
Hazard Nuker [ENHANCEMENT]

### ISSUE THREAD

**Issue by AreYouMehmetEfeisModder** [Rdimo/Hazard-Nuker#232]
A feature like mass dm should be brought, let's put my ids in a txt file and from there we can send messages to the people we want

**Comment by GodFather227** [comment]
Mass dm basically patched but aight

**Comment by AreYouMehmetEfeisModder** [comment]
I want to say that let's put the ids in the id.txt file and we can get the ids of the people on the server and send them a message. There are many such tools, but there is no one as fast as hazard nuker.

**Comment by GodFather227** [comment]
> [Deneme.zip](https://github.com/Rdimo/Hazard-Nuker/files/9149400/Deneme.zip)
> For example, there is this project, I add ids to data and when I run it, it sends messages in message.json to the ids, but it would be great if it came to Hazard Nuker because hazard nuker is very fast, this tool is very slow @Rdimo

Do you know why it’s slow?

**Comment by AreYouMehmetEfeisModder** [comment]
Because there is no proxy like hazard nuker's proxies, it would be great if you add this feature to hazard nuker, I have a tool in my hand, it takes the id of everyone on the server, and I put it in this tool. It sends a message to everyone on the discord server.

**Comment by GodFather227** [comment]
> Because there is no proxy like hazard nuker's proxies, it would be great if you add this feature to hazard nuker, I have a tool in my hand, it takes the id of everyone on the server, and I put it in this tool. It sends a message to everyone on the discord server.

No… things like mass dm go don’t work well is because of rate-limiting. This makes it’s very hard to automate a user token into doings things such as messaging others. Also they added captcha to messages if you send too many. Do even With proxies the token will still get rate-limited. And also I believe the best way to do it without getting rate limited is using an actual discord bot which shouldn’t be too hard to make yourself.

**Comment by AreYouMehmetEfeisModder** [comment]
ok, bring this feature, I think it will be useful for a lot of people Even if it is slow, it would be great to have Hazard Nuker in it. There is a tool that takes the ids for automation, but if you create an id.txt file, anyone can put it there and get rid of the automation.

**Comment by AreYouMehmetEfeisModder** [comment]
İ find this project can do there is such a thing as anti captha maybe you can do it 
https://github.com/ebankoff/overlord

**Comment by GodFather227** [comment]
> İ find this project can do there is such a thing as anti captha maybe you can do it
> https://github.com/ebankoff/overlord

If you figure that out and send it I’ll be more than happy to add it

**Comment by AreYouMehmetEfeisModder** [comment]
yes, he can pass capthas and send messages by using anti captha. It's very good to ids. I've been looking for this tool for 4 days.

**Comment by AreYouMehmetEfeisModder** [comment]
What do you mean solve?

**Comment by GodFather227** [comment]
> What do you mean solve?

Make a python anti captcha that works with discord and doesn’t use api keys where you have to pay for captcha support

**Comment by AreYouMehmetEfeisModder** [comment]
must have api

**Comment by GodFather227** [comment]
> must have api

And whose gonna pay for the anti captcha?

**Comment by AreYouMehmetEfeisModder** [comment]
I will try to find

**Comment by AreYouMehmetEfeisModder** [comment]
https://github.com/Ju0x/dc-captcha-methods look I don't know if it will work, I will try

**Comment by AreYouMehmetEfeisModder** [comment]
https://github.com/Vishalop491/DISCORD-CAPTCHA-JOINER this may work

**Comment by GodFather227** [comment]
> https://github.com/Vishalop491/DISCORD-CAPTCHA-JOINER this may work

Cool make pull

**Comment by AreYouMehmetEfeisModder** [comment]
İ dont know
